fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -293,13 +340,29 @@ if [ ! -e "$fMRIFolder" ] ; then
   log_Msg "mkdir ${fMRIFolder}"
   mkdir "$fMRIFolder"
 fi
-cp "$fMRITimeSeries" "$fMRIFolder"/"$OrigTCSName".nii.gz
+${FSLDIR}/bin/imcp "$fMRIScout" "$fMRIFolder"/"$OrigScoutName"
+
+# --- Do slice time correction if indicated
+if [ $DoSliceTimeCorrection = "TRUE" ] ; then
+    log_Msg "Running slice timing correction"
+    SliceTimerCorrectionParameters=`echo ${SliceTimerCorrectionParameters} | sed 's/@/ /g'` 
+    ${FSLDIR}/bin/slicetimer -i "$fMRIFolder"/"$OrigTCSName" -o "$fMRIFolder"/"$OrigTCSName"_orig_stc -r ${TR} -v ${SliceTimerCorrectionParameters}
```

Comment:
Please use the code pattern I pointed to for this, and try to avoid overwriting a variable's value, as it makes debugging harder:

```
IFS='@' read -a SliceTimerCorrectionParametersArray <<< "$SliceTimerCorrectionParameters"
${FSLDIR}/bin/slicetimer -i "$fMRIFolder"/"$OrigTCSName" -o "$fMRIFolder"/"$OrigTCSName"_orig_stc -r ${TR} -v "${SliceTimerCorrectionParametersArray[@]}"
```

Reply:
Changed it. I'm just weary whether it works on all systems and BASH versions.

Reply:
We already use it elsewhere:

https://github.com/Washington-University/HCPpipelines/blob/master/ICAFIX/ReApplyFixMultiRunPipeline.sh#L681

mac has the oldest bash version I know of for any non-ancient system, and it works there (though it is missing associative arrays and some nice things like `[[ -v <var ]]`).

Reply:
Better example: https://github.com/Washington-University/HCPpipelines/blob/master/DeDriftAndResample/DeDriftAndResamplePipeline.sh#L469